wmn-data.json

```diff
@@ -515,6 +515,19 @@
       ],
       "cat": "gaming"
     },
+    {
+      "name": "Arsmate",
+      "uri_check": "https://arsmate.com/{account}",
+      "e_code": 200,
+      "e_string": "og:title",
+      "m_string": "no encontrada</title>",
```

Comment:
I think this string may be dependent on your language setting. I'm using English and do not find it. Can you look for something like a page element that may not change? Something like `<title>404 `